### ISSUE TITLE
Can not drag around workflow editor if there is a large comment

### ISSUE THREAD

**Issue by ahmedhamidawan** [galaxyproject/galaxy#18764]
https://github.com/user-attachments/assets/b2500b55-655f-4eaf-bb49-97955b2c0735

Sample invocation I tested this on: https://usegalaxy.org/workflows/invocations/a11d3f54288f2b34?from_panel=true

**Comment by dannon** [comment]
@ElectronicBlueberry might have ideas here

**Comment by ElectronicBlueberry** [comment]
I feel like this is a UX question rather than a bug.

The behavior, as shown in the video, is currently intentional, so that text can be selected and copied out of comments.
In this scenario, it is a hindrance, rather than a benefit.

I see a few possible approaches here:

* Don't have a comments text selectable
* Don't intercept scroll on text comments, so it is easier to zoom out past the comment. This will however not work for markdown comments, since scrolling is sometimes needed there.
* Have a "copy text" button appear on comment hover, instead of having the text selectable.

**Comment by dannon** [comment]
Yep, definitely a UX question, I should have added more notes from the dev meeting discussion instead of just drive-by tagging ;)

Folks agreed it's not really a bug but can be a minor pain point if you have lots of text and keep accidentally selecting insead of dragging.

We *do* want it to be selectable -- this is useful for people who might need to copy a reference identifier out of a comment, etc. -- unselectable text can be frustrating and we want to avoid that.

I like your option 3 idea, super clear for the user and while you couldn't select just a single bit of text it'd at least make that text available to the clipboard?

One suggestion from the meeting was to make the text 'click to make selectable', so comment text by default is unselectable and dragging would work like usual until you single clicked into the text (without a drag), which flipped it to this editable/selectable mode (though obvs not editable for other users workflows) until focus was lost.    Or, maybe this toggle-ability could be driven by the button you mention for your third approach for more clarity?
